debezium-core/src/main/java/io/debezium/connector/common/BaseSourceTask.java

```diff
@@ -45,8 +45,10 @@
 public abstract class BaseSourceTask<P extends Partition, O extends OffsetContext> extends SourceTask {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(BaseSourceTask.class);
-    private static final long INITIAL_POLL_PERIOD_IN_MILLIS = TimeUnit.SECONDS.toMillis(5);
-    private static final long MAX_POLL_PERIOD_IN_MILLIS = TimeUnit.HOURS.toMillis(1);
+    private static final long INITIAL_POLL_PERIOD_IN_MILLIS_ = TimeUnit.SECONDS.toMillis(5);
+    Duration INITIAL_POLL_PERIOD_IN_MILLIS = Duration.ofMillis(TimeUnit.SECONDS.toMillis(5));
+    private static final long MAX_POLL_PERIOD_IN_MILLIS_ = TimeUnit.HOURS.toMillis(1);
+    Duration MAX_POLL_PERIOD_IN_MILLIS = Duration.ofMillis(TimeUnit.HOURS.toMillis(1));
```

Comment:
IMHO the old instancebe removed - not adding underscores. The new instances should be `private static final`